test/addons/openssl-client-cert-engine/test.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+const common = require('../../common');
+const assert = require('assert');
+const https = require('https');
+const fs = require('fs');
+const path = require('path');
+
+const engine = path.join(__dirname, '/build/Release/testengine.engine');
```

Comment:
I think this needs to use:
```javascript
`./build/${common.buildType}/testengine.engine`);
```
to be able to support debug and release build types.

Reply:
Ooh, good, thanks, fixed.